build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -354,7 +354,7 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
 
                 Console.WriteLine($"Copying package {fileName} to artifacts directory {Dirs.CorehostLocalPackages}.");
             }
-            foreach (var item in hostVersion.LatestHostPackages)
+            foreach (var item in hostVersion.LatestHostPackagesToValidate)
```

Comment:
It will be good to port this change (LatestHostPackagesToValidate) to master as well.